pkg/packagekit/updates.jsx

```diff
@@ -284,7 +284,7 @@ function updateItem(info, pkgNames, key) {
         pkgsTruncated.push(<span key="more">…</span>);
 
     if (pkgNames.some(p => p.name.startsWith("kpatch-patch")))
-        pkgsTruncated.push(<>{" "}<Badge>{_("patches")}</Badge></>);
+        pkgsTruncated.push(<React.Fragment key={`${key}-kpatches-spacer`}>{" "}<Badge key={key + "kpatches"}>{_("patches")}</Badge></React.Fragment>);
```

Comment:
Why doing the same thing with two different syntaxes? Once `${key}` and then `key +`. But also the `Badge` does not necessarily need a key

Reply:
Hmmm, I later realised I could just use `${key}`, dropped the badge key as React is happy about the fragment having a key.

Reply:
The later "label group" commit cleans this up, but please rebase this for a clean history. Indeed let's just drop the Badge key here.